### ISSUE TITLE
Error: 'LintCode' is imported from both

### ISSUE THREAD

**Issue by karabanovbs** [solid-software/solid_lints#176]
Error:
/root/.pub-cache/hosted/pub.dev/solid_lints-0.1.5/lib/src/lints/double_literal_format/double_literal_format_rule.dart:37:35: Error: 'LintCode' is imported from both 'package:analyzer/src/dart/error/lint_codes.dart' and 'package:custom_lint_core/src/lint_codes.dart'.

custom_lint version: "0.6.7"
solid_lints version: "0.1.5"
analyzer version: "6.7.0"

**Comment by karabanovbs** [comment]
```console
Building package executable... 
Built custom_lint:custom_lint.
/Users/boriskarabanov/.pub-cache/hosted/pub.dev/solid_lints-0.1.5/lib/src/models/rule_config.dart:2:1: Error: 'LintCode' is imported from both 'package:analyzer/src/dart/error/lint_codes.dart' and 'package:custom_lint_core/src/lint_codes.dart'.
import 'package:custom_lint_builder/custom_lint_builder.dart';
^^^^^^^^
/Users/boriskarabanov/.pub-cache/hosted/pub.dev/solid_lints-0.1.5/lib/src/utils/parameter_utils.dart:3:1: Error: 'LintCode' is imported from both 'package:analyzer/src/dart/error/lint_codes.dart' and 'package:custom_lint_core/src/lint_codes.dart'.
import 'package:custom_lint_builder/custom_lint_builder.dart';
^^^^^^^^
/Users/boriskarabanov/.pub-cache/hosted/pub.dev/solid_lints-0.1.5/lib/src/lints/avoid_unnecessary_type_assertions/avoid_unnecessary_type_assertions_rule.dart:48:37: Error: 'LintCode' is imported from both 'package:analyzer/src/dart/error/lint_codes.dart' and 'package:custom_lint_core/src/lint_codes.dart'.
  static const _unnecessaryIsCode = LintCode(
                                    ^^^^^^^^
/Users/boriskarabanov/.pub-cache/hosted/pub.dev/solid_lints-0.1.5/lib/src/lints/avoid_unnecessary_type_assertions/avoid_unnecessary_type_assertions_rule.dart:53:44: Error: 'LintCode' is imported from both 'package:analyzer/src/dart/error/lint_codes.dart' and 'package:custom_lint_core/src/lint_codes.dart'.
  static const _unnecessaryWhereTypeCode = LintCode(
                                           ^^^^^^^^
/Users/boriskarabanov/.pub-cache/hosted/pub.dev/solid_lints-0.1.5/lib/src/lints/double_literal_format/double_literal_format_rule.dart:37:35: Error: 'LintCode' is imported from both 'package:analyzer/src/dart/error/lint_codes.dart' and 'package:custom_lint_core/src/lint_codes.dart'.
  static const _leadingZeroCode = LintCode(
                                  ^^^^^^^^
/Users/boriskarabanov/.pub-cache/hosted/pub.dev/solid_lints-0.1.5/lib/src/lints/double_literal_format/double_literal_format_rule.dart:45:38: Error: 'LintCode' is imported from both 'package:analyzer/src/dart/error/lint_codes.dart' and 'package:custom_lint_core/src/lint_codes.dart'.
  static const _leadingDecimalCode = LintCode(
                                     ^^^^^^^^
/Users/boriskarabanov/.pub-cache/hosted/pub.dev/solid_lints-0.1.5/lib/src/lints/double_literal_format/double_literal_format_rule.dart:54:36: Error: 'LintCode' is imported from both 'package:analyzer/src/dart/error/lint_codes.dart' and 'package:custom_lint_core/src/lint_codes.dart'.
  static const _trailingZeroCode = LintCode(
                                   ^^^^^^^^
/Users/boriskarabanov/.pub-cache/hosted/pub.dev/solid_lints-0.1.5/lib/src/models/rule_config.dart:38:28: Error: 'LintCode' is imported from both 'package:analyzer/src/dart/error/lint_codes.dart' and 'package:custom_lint_core/src/lint_codes.dart'.
  LintCode get lintCode => LintCode(
                           ^^^^^^^^
/Users/boriskarabanov/.pub-cache/hosted/pub.dev/solid_lints-0.1.5/lib/src/utils/parameter_utils.dart:36:7: Error: 'LintCode' is imported from both 'package:analyzer/src/dart/error/lint_codes.dart' and 'package:custom_lint_core/src/lint_codes.dart'.
      LintCode(
      ^^^^^^^^


Failed to start plugins
The request analysis.setContextRoots failed with the following error:
RequestErrorCode.PLUGIN_ERROR
Bad state: Failed to start the plugins.
at:
#0      SocketCustomLintServerToClientChannel._checkInitializationFail.<anonymous closure> (package:custom_lint/src/v2/server_to_client_channel.dart:223:11)
<asynchronous suspension>
#1      Future.any.onValue (dart:async/future.dart:628:5)
<asynchronous suspension>

_PrettyRequestFailure: {"code":"PLUGIN_ERROR","message":"Bad state: Failed to start the plugins.","stackTrace":"#0      SocketCustomLintServerToClientChannel._checkInitializationFail.<anonymous closure> (package:custom_lint/src/v2/server_to_client_channel.dart:223:11)\n<asynchronous suspension>\n#1      Future.any.onValue (dart:async/future.dart:628:5)\n<asynchronous suspension>\n"}
#0      ChannelBase.sendRequest (package:custom_lint/src/server_isolate_channel.dart:82:7)
<asynchronous suspension>
#1      CustomLintRunner.initialize.<anonymous closure> (package:custom_lint/src/runner.dart:43:5)
<asynchronous suspension>
#2      _runServer.<anonymous closure> (package:custom_lint/custom_lint.dart:96:7)
<asynchronous suspension>
#3      _AsyncCompleter.complete (dart:async/future_impl.dart:41:3)
<asynchronous suspension>
#4      _runServer.<anonymous closure> (package:custom_lint/custom_lint.dart:126:19)
<asynchronous suspension>
#5      _runServer (package:custom_lint/custom_lint.dart:86:3)
<asynchronous suspension>
#6      customLint (package:custom_lint/custom_lint.dart:50:5)
<asynchronous suspension>
#7      entrypoint (file:///Users/boriskarabanov/.pub-cache/hosted/pub.dev/custom_lint-0.6.7/bin/custom_lint.dart:68:3)
<asynchronous suspension>
#8      main (file:///Users/boriskarabanov/.pub-cache/hosted/pub.dev/custom_lint-0.6.7/bin/custom_lint.dart:80:5)
<asynchronous suspension>
```

**Comment by illia-romanenko** [comment]
Fixed in 0.2.3
